e2e/__tests__/__snapshots__/coverage_report.test.js.snap

```diff
@@ -55,12 +55,12 @@ exports[`outputs coverage report 1`] = `
 "-------------------------------------|----------|----------|----------|----------|-------------------|
 File                                 |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
 -------------------------------------|----------|----------|----------|----------|-------------------|
-All files                            |    56.52 |        0 |       50 |    55.56 |                   |
- coverage-report                     |    41.18 |        0 |       25 |    42.86 |                   |
+All files                            |    59.09 |        0 |       50 |    58.82 |                   |
+ coverage-report                     |    43.75 |        0 |       25 |    46.15 |                   |
   OtherFile.js                       |      100 |      100 |      100 |      100 |                   |
   Sum.js                             |    85.71 |      100 |       50 |      100 |                   |
   SumDependency.js                   |        0 |        0 |        0 |        0 |           8,10,12 |
-  notRequiredInTestSuite.js          |        0 |        0 |        0 |        0 |     8,15,16,17,19 |
```

Comment:
this is the thing I think is a big - `8` is no longer "uncovered"

Reply:
Not just uncovered, it appears that it was not instrumented.